src/main/java/racingcar/util/style/DashDistanceStyle.java

```diff
@@ -0,0 +1,21 @@
+package racingcar.util.style;
+
+import java.util.stream.IntStream;
+
+public class DashDistanceStyle implements DistanceStyle {
+    private static final String STRING_ONE_DISTANCE = "-";
+
+    public DashDistanceStyle() {
+        // Empty
+    }
+
+    @Override
+    public String getDistanceString(int distance) {
+        final StringBuilder builder = new StringBuilder(distance);
+
+        IntStream.rangeClosed(1, distance)
+                .forEach(value -> builder.append(STRING_ONE_DISTANCE));
+
+        return builder.toString();
```

Comment:
기본 리터럴에도 반복 메소드를 사용할 수 있다.

```suggestion
    public String getDistanceString(int distance) {
        return STRING_ONE_DISTANCE.repeat(distance);
```